### PR TITLE
Update PendingCallback.java

### LINKED ISSUE #72: BadParcelableException: ClassNotFoundException for PendingCallback
```
12-19 17:25:22.014 18473-18473/? E/AndroidRuntime: FATAL EXCEPTION: main
                                                   Process: com.supercilex.robotscouter, PID: 18473
                                                   java.lang.RuntimeException: Unable to start service com.firebase.jobdispatcher.GooglePlayReceiver@61e66f2 with Intent { act=com.google.android.gms.gcm.ACTION_TASK_READY cmp=com.supercilex.robotscouter/com.firebase.jobdispatcher.GooglePlayReceiver (has extras) }: android.os.BadParcelableException: ClassNotFoundException when unmarshalling: com.google.android.gms.gcm.PendingCallback
                                                       at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3343)
                                                       at android.app.ActivityThread.-wrap21(ActivityThread.java)
                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1582)
                                                       at android.os.Handler.dispatchMessage(Handler.java:102)
                                                       at android.os.Looper.loop(Looper.java:154)
                                                       at android.app.ActivityThread.main(ActivityThread.java:6119)
                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
                                                    Caused by: android.os.BadParcelableException: ClassNotFoundException when unmarshalling: com.google.android.gms.gcm.PendingCallback
                                                       at android.os.Parcel.readParcelableCreator(Parcel.java:2536)
                                                       at android.os.Parcel.readParcelable(Parcel.java:2462)
                                                       at android.os.Parcel.readValue(Parcel.java:2365)
                                                       at android.os.Parcel.readArrayMapInternal(Parcel.java:2732)
                                                       at android.os.BaseBundle.unparcel(BaseBundle.java:269)
                                                       at android.os.Bundle.getParcelable(Bundle.java:864)
                                                       at com.firebase.jobdispatcher.GooglePlayReceiver.a(Unknown Source)
                                                       at com.firebase.jobdispatcher.GooglePlayReceiver.onStartCommand(Unknown Source)
                                                       at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3326)
                                                       at android.app.ActivityThread.-wrap21(ActivityThread.java) 
                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1582) 
                                                       at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                       at android.os.Looper.loop(Looper.java:154) 
                                                       at android.app.ActivityThread.main(ActivityThread.java:6119) 
                                                       at java.lang.reflect.Method.invoke(Native Method) 
                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886) 
                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

### PR DESCRIPTION
**Pull request by SUPERCILEX**
Fix #72

### CODE REVIEW

File: jobdispatcher/build.gradle [hunk 1]
```diff
@@ -22,10 +22,6 @@ android {
         debug {
             testCoverageEnabled true
         }
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
```
Comment (ciarand) [inline jobdispatcher/build.gradle]: Why does removing this fix the issue? Is your app obfuscating the PendingCallback? What are your proguard settings?

**Comment by SUPERCILEX** [comment]
@ciarand I forgot that I had a PR going for the branch. The fix doesn't work anyway, I'm still trying to figure it out.

**Comment by SUPERCILEX** [comment]
3rd time's a charm! @ciarand This is an urgent fix, so could you please release `v0.5.2`? I'm using [jitpack.io](https://www.jitpack.io) to work around the issue, but it would be nice to have the fix buit in.

For anyone looking to use my fix, include: `compile 'com.github.SUPERCILEX.firebase-jobdispatcher-android:firebase-jobdispatcher:patch-1-SNAPSHOT'`

File: common/constants.gradle [hunk 2]
```diff
@@ -3,8 +3,8 @@ project.ext {
     projectName = "firebase-jobdispatcher-android"
     group = "com.firebase"
     version = "0.5.1"
-    buildtools = "25.0.0"
-    supportLibraryVersion = "25.0.0"
+    buildtools = "25.0.2"
+    supportLibraryVersion = "25.1.0"
```
Comment (ciarand) [inline common/constants.gradle]: Do we need to bump these versions as well as include the `@Keep` annotation? I'd rather limit the change to the minimum of what's required to fix the issue.

**Comment by ciarand** [comment]
I'd be happy to take a PR that just includes 234705294b9bac90ce901448a6535683dfb36ce5. You can either remove the other changes, or if you're in a hurry, you can just leave a comment in #74 stating you're ok with your commits being submitted to the project and I'll submit that and cut a release.

**Comment by SUPERCILEX** [comment]
@ciarand #74 is fine.

**Pull request closed without merging**